assembly/common-rules/src/main/java/org/eclipse/sw360/antenna/rules/AntennaCommonRulesPackage.java

```diff
@@ -13,14 +13,16 @@
 
 import org.eclipse.sw360.antenna.api.IRulesPackage;
 
+import java.net.URISyntaxException;
+
 public class AntennaCommonRulesPackage implements IRulesPackage {
     @Override
     public String getRulesPackageName() {
         return "Antenna Common Rules";
     }
 
     @Override
-    public String getRulesetFolder() {
-        return getClass().getProtectionDomain().getCodeSource().getLocation().getPath();
+    public String getRulesetFolder() throws URISyntaxException {
+        return getClass().getProtectionDomain().getCodeSource().getLocation().toURI().getPath().replaceFirst("/", "");
```

Comment:
the part `.replaceFirst("/", "")` looks very dirty and unsafe.

Reply:
I think the same thing, that's why this is WIP. 
I wasn't able to get a better solution than that for windows, since neither .toUri(), nor .getPath(), nor .getFile() returned a String that didn't cause an `InvalidPathExcpetion` to occur. 

I didn't find anything better yesterday. Will go on a new google excursion this morning.